File: Arrays/Python/Intersection_of_two_arrays.py
class Solution:
    def NumberofElementsInIntersection(self,a, b, n, m):
        #return: expected length of the intersection array.
        
        #code here
        a = set(a)
        b = set(b)
        a.intersection_update(b)
        c = len(list(a))
        return c

File: Arrays/Java/Intersection_of_two_arrays.java
class Solution {
    // Function to return the count of the number of elements in
    // the intersection of two arrays.
    public static int NumberofElementsInIntersection(int a[], int b[], int n, int m) {
        // Your code here
        HashSet<Integer> set = new HashSet<>();
        for (int i:a) {
            set.add(i);
        }
        int c=0;
        for (int j:b) {
            if (set.contains(j)) {
                c++;
                set.remove(j);
            }
        }
        return c;
    }
};
